nncf/quantization/algorithms/min_max/torch_backend.py

```diff
@@ -141,10 +134,10 @@ def create_quantizer_insertion_command(
     def unify_statistics(statistics: List[PTMinMaxTensorStatistic]) -> PTMinMaxTensorStatistic:
         max_values, min_values = [], []
         for statistic in statistics:
-            max_values.append(torch.tensor(statistic.max_values).flatten())
-            min_values.append(torch.tensor(statistic.min_values).flatten())
-        max_values = torch.max(torch.tensor(max_values))
-        min_values = torch.min(torch.tensor(min_values))
+            max_values.append(statistic.max_values.flatten())
+            min_values.append(statistic.min_values.flatten())
+        max_values = torch.amax(torch.stack(max_values), dim=0)
+        min_values = torch.amin(torch.stack(min_values), dim=0)
```

Comment:
I don't get why this moment wasn't placed into the common (algo) part, while the `FastBiasCorrection` parts were.